Atomic/storage.py

```diff
@@ -184,7 +184,11 @@ def modify(self):
                         cmd = 'bash -c ". /usr/lib/docker-storage-setup/libdss.sh; check_data_size_syntax {0}"'.format(self.args.lvsize)
                         util.check_call(cmd)
                     except subprocess.CalledProcessError:
-                        raise ValueError("Invalid format for --lvsize")
+                        try:
+                            cmd = 'bash -c ". /usr/share/container-storage-setup/libcss.sh; check_data_size_syntax {0}"'.format(self.args.lvsize)
```

Comment:
There are many references to d-s-s in the code that should be updated, I just added that one since it was "gating" the test right now. I think we should take a deeper look into reworking some of the storage code as well, since there seems to be some weird interaction with devmapper vs overlay, and I'm not sure if the storage code is using c-s-s correctly.